include/mme-app/utils/mmeProcedureTypes.h

```diff
@@ -33,6 +34,8 @@ namespace mme
     	tau_c,
     	s1Handover_c,
         erabModInd_c,
+	cbReq_c,
```

Comment:
wondering what is difference between  CBReq_c & dedBrActivation_c procedure?

Reply:
CBReq_c is at the session level and it anchors the activation of dedicated bearers, as each bearer gets activated at a different pace. At the end of activation of dedicated bearers, dedBrActivation_c informs CBReq_c , which sends the Create Bearer Response.